.github/workflows/build-checks.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
         if: ${{ matrix.component != 'runtime' }}
         run: sudo apt-get -y install musl-tools
       - name: Install devicemapper
-        if: ${{ matrix.command == 'make check' && matrix.component == 'agent' }}
+        if: ${{ (matrix.command == 'make check' || matrix.command == 'make test') && matrix.component == 'agent' }}
```

Comment:
Optional suggestion:
I wonder if we could make the commit message be a bit cleaner for future references and explain why it's needed.

e.g. something like (but probably more accurate than my understanding):

> checks: Install dependency of agent unit tests
> 
> With the requirement of devicemapper in the agent as part of the
> work to add integrity checks of shared images in the agent
> we now require `libdevmapper-dev` to be installed in order
> to run the agent unit tests

?